airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -370,7 +370,10 @@ def read_records(
             raise e
 
     def parse_response_error_message(self, response: requests.Response) -> Optional[str]:
-        body = response.json()
+        try:
```

Comment:
what's an example response currently causing this to fail? I wonder if we an parse it properly. What endpoint is failing for example?